services/pushover/config.go

```diff
@@ -0,0 +1,57 @@
+package pushover
+
+import (
+	"net/url"
+
+	"github.com/pkg/errors"
+)
+
+// DefaultPushoverURL is the default URL for the Pushover API.
+const (
+	DefaultPushoverURL = "https://api.pushover.net/1/messages.json"
+)
+
+// Config is the [pushover] configuration as defined in the Kapacitor configuration file.
+type Config struct {
+	// Whether Pushover integration is enabled.
+	Enabled bool `toml:"enabled" override:"enabled"`
+	// The Pushover API token.
+	Token string `toml:"token" override:"token,redact"`
+	// The User/Group that will be alerted.
+	UserKey string `toml:"user_key" override:"user_key,redact"`
```

Comment:
One last small change. Can this be `user-key` instead? The Kapacitor config option name convention is all over the place but currently the `-` is winning so I have decided that should be the standard going forward.